pages/specialtiesPage.ts

```diff
@@ -7,12 +7,41 @@ export class SpecialtiesPage extends HelperBase{
         super(page)
     }
 
+    async validateExistenceOf(specialty: string){
+        const entries = await this.page.getByRole('textbox').all()
+
+        for(let entry of entries){
+            if(specialty === await entry.inputValue()){
+                return true
+            }
+        }
+        return false
```

Comment:
This is not a good implementation because when a test fails, you will see something like "Expected true but actual false" in the error message. Not descriptive at all. Also in the test level it also look not very nice, when you pass the page object method as argument into the expect assertion.
Just make this validation inside of this method and make method be responsible for the validation. It will make test looks cleaner and assertion error message more meaningful when test will fail.
P.S. you can use `toContainEqual()` assertion to validate if your array of values has expected value or not

Reply:
Thanks for this feedback!  It's super helpful.  I was able to re-work a lot of it, but have some remaining questions that I will post in Slack channel.